FRW/UI/Component/PersistentToastView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PersistentToastView.swift
+//  FRW
+//
+//  Created by Antonio Bello on 11/11/24.
+//
+
+import SwiftUI
+
+// Note: this seems to be a clone of `CalloutView`, but in the opposite vertical direction.
+struct PersistentToastView: View {
+    let message: String
+    let imageRes: ImageResource
+    
+    var body: some View {
+        HStack {
+            HStack(spacing: 8) {
+                Image(self.imageRes)
+                    .resizable()
+                    .frame(width: 20, height: 20)
+                Text(self.message)
+                    .foregroundColor(Color(.orange1))
+                    .font(.inter(size: 12, weight: .w400))
+                Spacer()
+            }
+            .padding(.vertical, 8)
+            .padding(.horizontal, 12)
+            .background(Color(.orange5))
+        }
+        .background(Color.LL.Neutrals.background.opacity(0.95))
```

Comment:
I think the opacity is not the same as design

Reply:
The opacity is defined in the orange5 color. I added the background at line 30 to control transparency, as the content behind the view would make it unreadable. Compare the two:

![image](https://github.com/user-attachments/assets/dded03f7-64aa-4f05-ba2c-d61c8263adbe)

![image](https://github.com/user-attachments/assets/2c7b0bce-39ea-444f-81ba-ecfb7458b388)

The alternative is to make sure the content behind doesn't scroll, but that would require the sheet to be taller. Suggestions?

Reply:
ok， @KK-UIUX  Can we review this in UI ?